zeekpkg/package.py

```diff
@@ -85,42 +86,6 @@ def short_description(metadata_dict):
     return rval.lstrip()
 
 
-def user_vars(metadata_dict):
```

Comment:
Don't know how likely it is someone relies on this API, but since it's not originally marked as internal in any way, I'd just leave it be an alias for `UserVar.parse_dict`.

Reply:
Whoops! Yes, thanks for catching this.

Reply:
Agreed, yeah. Will do.